langs/jq/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM alpine:3.20 AS builder
+
+ARG RELEASE=1.7.1
```

Comment:
I tend to only use ARG for variables that need to pass across multiple stages, just use VAR here. Also for consistency we tend to call this variable VER

Reply:
So, there's `VAR` too, awesome stuff. Alright, I'll get that done, both here and APL.

Reply:
> So, there's `VAR` too, awesome stuff. Alright, I'll get that done, both here and APL.

No, there isn't. 😅 All good, though.